handsontable/src/pluginHooks.js

```diff
@@ -1427,25 +1427,31 @@ const REGISTERED_HOOKS = [
    * the selected cells. This hook is fired when {@link Options#copyPaste} option is enabled.
    *
    * @event Hooks#beforeCut
-   * @param {Array[]} data An array of arrays which contains data to cut.
-   * @param {object[]} coords An array of objects with ranges of the visual indexes (`startRow`, `startCol`, `endRow`, `endCol`)
-   *                       which will be cut out.
+   * @param {object} clipboardData Information about cut action which is going to happen.
```

Comment:
I think that it's worth mentioning that "Since @14.0.0...." the argument was changed.

Reply:
Done within: https://github.com/handsontable/handsontable/pull/10517/commits/1a162c122322d93646f2798674022f8e647d5113.